test/e2e_node/numa_alignment.go

```diff
@@ -1,273 +0,0 @@
-/*
```

Comment:
Why do we need to move this package into e2e test framework(`test/e2e/framework/`)?
Those functions are called from test/e2e_node tests only, so it is nice to keep it here for related functions.

Reply:
The only reason is I thought `test/e2e/framework/node` was the correct place where to put reusable code. I'll move as you suggested